convene-web/app/javascript/src/custom_components/room_card.scss

```diff
@@ -12,25 +12,26 @@
 
 
     footer {
-      @apply w-full flex items-center justify-center border-t border-gray-200;
+      @apply w-full flex flex-col items-center justify-center border-t border-gray-200;
 
-      .room-door_enter {
-        @apply flex justify-center text-sm leading-5 text-gray-700 font-medium border border-transparent rounded-br-lg transition ease-in-out duration-150;
+      .room-door_enter, .room-door_configure {
```

Comment:
Not blocking:

What I am inferring from the CSS class here is that we have a `room-door` block that has a `configure` element. Is that accurate? Or do we have a `room` block that has a `door` block that has a `configure` element?

As an aside, It seems like we are not consistently doing either `.block_element--modifier` as a single class for targeting elements or if we are using  `.block .element.--modifer` syntax.

On the one hand, the `.block_element--modifier` syntax makes it really specific; however it means that we have to generate/write CSS for every block and every element and every modifier permutation. That likely will give us  near pixel-perfect control; and mitigate selector specificity hell; but it will also result in incredibly large CSS files and rule-sets, which  (over time) may reduce the performance and accessibility of the HTML/CSS rendering.

On the flip side; `.block .element .--modifier` means we can rely on `.--warning { background-color: red }` to provide a consistent experience, _so long as_ there is not a specificity conflict (i.e. `.block .element { background-color: blue }` would have higher specificity, and therefore take precedence.

I'd be curious what senior front-end developers think about the alternatives, and maybe we can check with Ambreen or Betsy think about the tradeoffs there?

Reply:
Still need some time to digest this.